### ISSUE TITLE
6.2.3 regression: command 'sort-imports.sort' not found

### ISSUE THREAD

**Issue by OliverJAsh** [amatiasq/vsc-sort-imports#78]
```
Command 'Sort imports' resulted in an error (command 'sort-imports.sort' not found)
```

![image](https://user-images.githubusercontent.com/921609/107378531-f7a77780-6ae3-11eb-91ef-69b1648d0faa.png)

I downgraded to 6.2.2 and the problem disappeared.

**Comment by jerome-benoit** [comment]
Will be fixed in the upcoming 6.2.4 version. 

Thanks.

**Comment by kmcfate** [comment]
This is still not fixed in 6.2.4.

**Comment by jerome-benoit** [comment]
> This is still not fixed in 6.2.4.

Of course it is. That version is reliabily working since months for all users (and they are quite a few ;))
